certified-connectors/Office365GroupsMail/apiProperties.json

```diff
@@ -0,0 +1,38 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions",
+      "triggers"
+    ],
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "<<Client ID placeholder>>",
+          "scopes": [
+            "Group.ReadWrite.All"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "AzureActiveDirectoryResourceId": "https://graph.microsoft.com"
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://graph.microsoft.com/"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#EB3C00",
+    "publisher": "Microsoft Corporation"
```

Comment:
For consistenacy, lets use Microsoft

Reply:
Got it, I saw others using `Microsoft Corporation` (e.g. AzureIoTCentral and Partner Center Referrals), I'll change it in the next iteration :)